labels/ethereum/mooniswap_and_1inch_pools.sql

```diff
@@ -0,0 +1,66 @@
+-- https://duneanalytics.com/queries/87581
+-- 1inch
+WITH pools_1inch AS (
+    SELECT
+        token1,
+        token2,
+        mooniswap
+    FROM onelp."MooniswapFactory_evt_Deployed"
+    UNION ALL
+    SELECT
+        token1,
+        token2,
+        mooniswap
+    FROM onelp."MooniswapFactory_v2_evt_Deployed"
+),
+pools_1inch_first_token AS (
+SELECT
+    mooniswap AS pool_address,
+    t.symbol as token_0_symbol
+FROM onelp."MooniswapFactory_evt_Deployed" dex
+INNER JOIN erc20.tokens t ON dex.token1 = t.contract_address
+WHERE t.symbol IS NOT NULL
+),
+pools_1inch_second_token AS (
+SELECT
+    mooniswap AS pool_address,
+    t.symbol as token_1_symbol
+FROM onelp."MooniswapFactory_evt_Deployed" dex
+INNER JOIN erc20.tokens t ON dex.token2 = t.contract_address
+WHERE t.symbol IS NOT NULL
```

Comment:
Doesn't the inner join take care of this? That is, only results that match are returned.